packages/pulumi-aws/src/apps/storage/StorageElasticSearch.ts

```diff
@@ -0,0 +1,243 @@
+import path from "path";
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import { defineAppModule, PulumiApp } from "@webiny/pulumi-sdk";
+
+import { getAwsAccountId } from "../awsUtils";
+
+export interface ElasticSearchParams {
+    protect: boolean;
+}
+
+export const ElasticSearch = defineAppModule({
+    name: "ElasticSearch",
+    config(app, params: ElasticSearchParams) {
+        const domainName = "webiny-js";
+        const accountId = getAwsAccountId(app);
+
+        const domain = app.addResource(aws.elasticsearch.Domain, {
+            name: domainName,
+            config: {
+                elasticsearchVersion: "7.7",
+                clusterConfig: {
+                    instanceType: "t3.medium.elasticsearch",
+                    instanceCount: 2,
+                    zoneAwarenessEnabled: true,
+                    zoneAwarenessConfig: {
+                        availabilityZoneCount: 2
+                    }
+                },
+                // TODO: vpc is currently only in API
```

Comment:
Is this comment relevant? I'd say yes 🤔 

ES needs to be deployed into a VPC if `vpc` was set to true.